### PR TITLE
Displaying Notifications and Received Messages

### LINKED ISSUE #145: Receive Messages
**Is your feature request related to a problem? Please describe.**
User wants to receive Messages as this is a Messenger application :)

**Describe the solution you'd like**
New Messages should be stored in the chat they belong to.

Estimate Pizza: 4

### PR DESCRIPTION
**Pull request by rohansaw**
We can now display push notifications and show received messages. We also added a better Test Setup.
closes #145

### CODE REVIEW

File: .smalltalk.lint.ston [hunk 1]
```diff
@@ -1,18 +0,0 @@
-  
-SmalltalkCISpec {
```
Comment (schmidtjonas) [inline .smalltalk.lint.ston:2]: Why is this file deleted? Why is the reader so different? Is this intended?

Comment (Lasklu) [reply .smalltalk.lint.ston:2]: looks like squeak has deleted it while merging: [Commit](https://github.com/hpi-swa-teaching/TelegramClient/pull/194/commits/938239b7f461628de278274f05eaf149d9d43131?file-filters%5B%5D=.md&file-filters%5B%5D=.st&file-filters%5B%5D=.ston&file-filters%5B%5D=.yml)
**Resolved**
